django/river/settings/dev.py

```diff
@@ -6,6 +6,11 @@
 
 SECRET_KEY = "USE_IN_DEV_ONLY"
 
+INSTALLED_APPS += ["drf_spectacular"]
```

Comment:
sorry why is this used only in dev again ?

Reply:
I'm using this to generate the openapi schema: https://github.com/tfranzel/drf-spectacular
It extends the drf schema generator. Here I need it to type my SerializerMethodField. It can also split the request and response schemas. so I have more precise typing in the frontend. It also supports drf camelcase if we decide to have it in the future.